main.js

```diff
@@ -1,28 +1,44 @@
 var blink = require('./flash.js');
 
+var controlLed = DigitalOut(LED1);
+
 var ble = BLEDevice();
-var batterySrvcUUID = '180f';
-var batteryCharUUID = '2a19';
-var batteryChar = BLECharacteristic(batteryCharUUID, ['read', 'notify'], 1);
-var batteryService = BLEService(batterySrvcUUID, [batteryChar]);
+var batteryChar = BLECharacteristic('2a19', ['read', 'notify'], 1);
```

Comment:
It used to be as separate variables because there was no `getUUID()` function on the service, thus the UUID was used twice. Now there isn't anymore. But I'm open to changing it.